controllers/articles.controllers.js

```diff
@@ -8,3 +8,12 @@ exports.getArticle = (request, response, next) => {
 		})
 		.catch(next)
 }
+
+exports.getArticles = (request, response, next) => {
+	getAllArticles().then((articles) => {
+		articles.forEach((article) => {
+			article.comment_count = +article.comment_count
```

Comment:
Rather than converting the comment_count to a number in your controller, we should really be doing any data manipulation in the model. Could you please refactor so this is done there instead? If you take a look at CAST in SQL, it should help you out 🙂